src/main/java/uk/gov/justice/hmpps/datacompliance/repository/jpa/model/referral/ReferredOffenderIds.java

```diff
@@ -26,8 +26,8 @@
 @AllArgsConstructor
 @EqualsAndHashCode(of = {"referredOffenderBookingId"})
 @ToString(exclude = "offenderDeletionReferral") // to avoid circular reference
-@Table(name = "REFERRED_OFFENDER_BOOKING")
-public class ReferredOffenderBooking {
+@Table(name = "REFERRED_OFFENDER_IDS")
+public class ReferredOffenderIds {
```

Comment:
why is this plural?  Isn't it just holding a single link to an offender and booking?

Reply:
It reads like it holds a list of referred offender IDs but each row in the table is an offender record with an optional booking record

Reply:
The `REFERRED_OFFENDER_BOOKING_ID` column sounds like it is the offender booking id relating to the referral, but should really be renamed to match the new table name

Reply:
I've renamed to `ReferredOffenderAlias`.  Really I probably should have had a separate table just to store the list of booking ids for a given offender id to keep normal form, but that seemed a bit overkill at the time just to store one id.